dts/arm/microchip/mec1501hsz.dtsi

```diff
@@ -219,6 +219,17 @@
 			#size-cells = <0>;
 			status = "disabled";
 		};
+		espi_saf0: espi@40008000 {
+			compatible = "microchip,xec-espi-saf";
+			reg = < 0x40008000 0x400
+				0x40070000 0x400
+				0x40071000 0x400>;
+			label = "ESPI_SAF_0";
+			#address-cells = <1>;
```

Comment:
why are those commented out?

Reply:
> why are those commented out?

Those aren't comments. It's a property named `#address-cells`.